gradle.properties

```diff
@@ -19,4 +19,23 @@ android.useAndroidX=true
 # Kotlin code style for this project: "official" or "obsolete":
 kotlin.code.style=official
 
-detektVersion=1.18.0-RC2
\ No newline at end of file
+detektVersion=1.18.0-RC2
+
+
+POM_NAME=PayPal Payments - Android
+POM_PACKAGING=aar
+POM_DESCRIPTION=Paypal payment methods for Android
+POM_URL=not_defined
+
+POM_SCM_URL=https://github.com/paypal/Android-SDK
+POM_SCM_CONNECTION=scm:git:git://paypal/Android-SDK.git
+POM_SCM_DEV_CONNECTION=scm:git:ssh://paypal/Android-SDK.git
+
+POM_LICENCE_NAME=not_defined
+POM_LICENCE_URL=not_defined
```

Comment:
Are these going to be addressed in a separate PR? Let's define them now if we can

Reply:
This is the one we have in our checkout repo: https://developer.paypal.com/docs/limited-release/native-checkout/license/
Im not sure if it applies though. Perhaps we can add TODO and create a ticket for it until we are sure?